contacts/models.py

```diff
@@ -8,6 +9,7 @@ class Contact(models.Model):
             message="Phone number must be entered in the format: '+999999999'. Up to 20 digits allowed."
     )
 
+    user = models.ForeignKey(User, null=True, blank=True)
```

Comment:
this is fine, but i wud like to have a separate app for user with its model having fields:
first_name, last_name, email, phone, address etc. but lets do it in separate branch after this gets merged!

secondly, a contact should never be created without a user.